packages/components/src/components/optgroup/optgroup.ts

```diff
@@ -83,7 +86,7 @@ export default class SdOptgroup extends SolidElement {
             <span>${this.label}</span>
           </slot>
         </div>
-        <div role="group" part="options">
+        <div role="listbox" part="options" aria-label="${this.localize.term('optionsList')}">
```

Comment:
Isn't this more an "option-group"? Because the whole thing is already a list, but the opt-group makes... groups in it, but not new lists?

Reply:
Fixed